### DISCUSSION TITLE
How can I make the title bar taller?

### DISCUSSION THREAD

**Discussion by HuyHung1408** [pyinstaller/pyinstaller#6717] [Help] [answered]
Here's the result after converted using Pyinstaller:
![image](https://user-images.githubusercontent.com/86362423/161361710-c005b5a0-b0eb-48c1-85d3-bedd6e495ce1.png)

Run from VS code:
![image](https://user-images.githubusercontent.com/86362423/161361726-5c14fb56-a0a7-4762-aac5-d5a6c930b2f0.png)

**Comment by rokm** [comment, marked as answer]
What version of Windows is that? What UI toolkit are you using? Can you provide a minimal example that produces different title bar when ran from VS code vs. when ran as a PyInstaller-frozen application?

**Reply by HuyHung1408** [reply]
That's Windows 11, I'm using Tkinter! You see the title bar height. You can see the different when looking closely to the title bar!
After converted to exe:
![image](https://user-images.githubusercontent.com/86362423/161565107-b02141b1-a120-4104-8254-817b8a9e9b03.png)

Running from VS code:
![image](https://user-images.githubusercontent.com/86362423/161565530-9a5af962-cc5c-42fb-b1b4-4734db558add.png)

**Reply by rokm** [reply]
Hmm, the basic tkinter program:
```python
import tkinter as tk
window = tk.Tk()
greeting = tk.Label(text="Test")
greeting.pack()
window.mainloop()
``` 
on Windows 11, python 3.10.3 and PyInstaller 4.10, produces executable that has same-height title bar as original python script (and same as other programs, such as command prompt's terminal window).

How does the above program behave on your machine when frozen?

**Reply by HuyHung1408** [reply]
It just the same as the result that I have shown you :(

**Reply by bwoodsend** [reply]
Can you confirm that running your code from `cmd` gives the same outcome as running from VS code?

**Reply by HuyHung1408** [reply]
Restarted the machine, looks good now, thanks!
